### PR TITLE
Remove non-code files

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

medic/cht-core#7413

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
Sweet! nice work on this. So so nice to see all the deletes in there :sunglasses: 

A couple of requests per my comments.

As well, these were deleted, but I could not found them added in another PR:
* [scripts/bump_rev.js](https://github.com/medic/cht-core/pull/7414/files#diff-13928653a1aa78961e86d5d260f2300e4e3a1aaf937c9d1272a2e59dc414cc23) 
* [scripts/fetch_contact_summary.sh](https://github.com/medic/cht-core/pull/7414/files#diff-01b58b30c972c5b5cd1482bdbb71b025deba23654dd00c2d88e5d3625d1399f7)
* [scripts/first_load_with_view.js](https://github.com/medic/cht-core/pull/7414/files#diff-c8a98336cc2fdc7c1ab621e7c3b14af057d7a8d17cd21669df5d818e50b6af25)
* [scripts/migrate-settings.js](https://github.com/medic/cht-core/pull/7414/files#diff-c198b53a33e57fa790a62a54892361e9745a09b6332f82ff63569b1b36bcb570)
* [scripts/po-editor-importer/import_po_editor.js](https://github.com/medic/cht-core/pull/7414/files#diff-456aaccfac2ef07de8b488beddf1409d9487e36ed61bc7f6e8742e7b2a1b3b2f)
* [scripts/po-editor-importer/readme.md](https://github.com/medic/cht-core/pull/7414/files#diff-4666f1e9c78a5f4f120affe18cc6bb96864faf4cabaec5c6c498ee7f49ab5462)
* [scripts/po-editor-importer/token_and_project.json.example](https://github.com/medic/cht-core/pull/7414/files#diff-d5b7f9dc6ba394c2423fe2bee814a2404dd0eca655305988636f6f3c889143c6)
* [scripts/replicate-db-per-user](https://github.com/medic/cht-core/pull/7414/files#diff-53744d9adb74333e45cc39519bcafe9c7c6544a415abdfa494a3e474599e1133)
* [scripts/rules/fetch_rules.sh](https://github.com/medic/cht-core/pull/7414/files#diff-c69817165f4dfa1638861e2367cbb484a4e403d30b1c1eb106cb89e8ddb715e9)
* [scripts/rules/update_tasks.sh](https://github.com/medic/cht-core/pull/7414/files#diff-a88b925fde9649c586b312ab03d1778ee3f27b35c007f768841c3deb1cb0d07c)
* [scripts/sms-report.js](https://github.com/medic/cht-core/pull/7414/files#diff-b7b22db6df24bd282a56c29b7a7570ad7144811b4bdc9088829c30aacca256d1)
* [scripts/translation_check](https://github.com/medic/cht-core/pull/7414/files#diff-055e27b9eccb4f253f880987aca1a29fcf521c903334331249f7baa829bd3650)
* [scripts/upload_contact_summary.sh](https://github.com/medic/cht-core/pull/7414/files#diff-a4c67fc46cd766502a6cd514beed7c572f98da23595445eab5325106dddcce07)
* [scripts/validate_phone_number.js](https://github.com/medic/cht-core/pull/7414/files#diff-9509bf454ac8ebfdac95b6569aa68aba17e925e7fde6aca806f85ed95081f9ff)

File: tests/scripts/e2e-servers.js [hunk 1]
```diff
@@ -124,13 +124,19 @@ app.post('/:server/:action', (req, res) => {
...
+      () => stopServer('sentinel'),
+    ])
+    .then(() => {
+      res.status(200).end();
+      process.exit(0);
+    })
+    .catch(() => {
+      res.status(200).end();
+      process.exit(0);
+    });
```
[the remark this answers is not in the document]
Comment (garethbowen) [reply tests/scripts/e2e-servers.js:139]: Actually, it was intentional! When I moved directories different eslint rules applied, which meant we needed a `.catch` here. It's almost certainly a waste of time as the whole point of this block is to exit the process! But in the interests of keeping eslint happy, are you ok with it?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,103 +1 @@
...
-- A detailed description of the steps required to reproduce the vulnerability (proof of concept source code, screenshots, and compressed screen captures are all helpful to us); and
-- Your name/handle and a link for recognition in our Hall of Fame.
-
-### Code of Conduct
-
-All maintainers and contributors are required to act according to our [Code of Conduct](https://github.com/medic/medic/blob/master/CODE_OF_CONDUCT.md). Thank you for your help building a positive community and a safe environment for everyone.
-
-#### License
-The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.
+Read about how to contribute on the [documentation site](https://docs.communityhealthtoolkit.org/contribute/).
```
[the remark this answers is not in the document]
Comment (garethbowen) [reply CONTRIBUTING.md:1]: The file I'm replacing talked about contributing documentation and so on, so to keep it consistent I thought it'd be better to link to the higher level and then people can choose code if that's all they're interested in.

**Comment by garethbowen** [comment]
Regarding the deleted scripts, I dropped them on purpose for various reasons:

- bump_rev.js: trivial and ancient
- fetch_contact_summary.sh: trivial and ancient
- first_load_with_view.js: ancient and dangerous
- migrate-settings.js: trivial and ancient
- po-editor-importer/*: obsolete - we have a new poe script
- replicate-db-per-user: unsafe
- rules/fetch_rules.sh: trivial and ancient
- rules/update_tasks.sh: obsolete and ancient
- sms-report.js: obsolete - admin app functionality replaces this
- translation_check: obsolete - poe scripting does this
- upload_contact_summary.sh: obsolete - use cht-conf
- validate_phone_number.js: trivial

File: tests/scripts/e2e-servers.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (mrjones-plip) [reply tests/scripts/e2e-servers.js:139]: Ah - yup yup! It's redundant code and non-critical.

File: CONTRIBUTING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mrjones-plip) [reply CONTRIBUTING.md:1]: cool cool - don't feel strongly either way.

**Review by mrjones-plip — APPROVED** [review]
After addressing minor discussion points, lgtm!  Be sure other two related PRs are ready to go to, obvs.

**Pull request merged**